project-addons/custom_partner/partner_view.xml

```diff
@@ -404,6 +404,7 @@
             <field name="arch" type="xml">
                 <field name="lang" position="after">
                     <field name="growth_rate"/>
+                    <label for="average_margin"/><div><field name="average_margin" class="oe_inline"/> %% </div>
```

Comment:
Es posible que necesitéis ocultar el campo a cierto grupo, por ejemplo "salesman" ya que a otros perfiles puede no interesarles esto y porque lo tenéis como query sino les fallaría sino tiene acceso a facturas por ejemplo.